src/core/LoadBalancer.ts

```diff
@@ -24,13 +24,12 @@ import {
 } from './MembershipListener';
 
 /**
- * {@link LoadBalancer} allows you to send operations to one of a number of endpoints (Members).
+ * A {@link LoadBalancer} allows you to send operations to one of a number of endpoints (Members).
```

Comment:
How about something like: "A LoadBalancer selects one of the endpoints (members) for the next send operation when the operation is not directed to a specific endpoint."

Reply:
Can a user direct an operation to a specific member? I think "when the operation is not directed to a specific endpoint." is internal detail and user does not need to know it. 

I checked grammarly it says "one of a number of endpoints" can be improved to be "one of many/some/several endpoints".

BTW, this explanation is the same of Java at the moment, maybe we should change that too.

I propose to use:

"A LoadBalancer allows you to send operations to one many endpoints (Members)." changing only "one of a number of" part

Reply:
The load balancer selects a member, not send operations to it. I think mentioning that the load balancer is only used in some cases is useful information, but if you'd like to left it out, the doc becomes even simpler, how about: "A LoadBalancer selects one of the endpoints (members) for the next send operation"

Reply:
A {@link LoadBalancer} selects one of the endpoints(Members) for the next send operation that be directed to many endpoints.

What about this? Did not choose your version since a user cannot direct an operation to an endpoint directly, but they can understand that some operations need to go to a specific member with this sentence.